src/foam/nanos/approval/ApprovalRequestClassificationEnum.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.ENUM({
+  package: 'foam.nanos.approval',
+  name: 'ApprovalRequestClassificationEnum',
+
+  values: [
+    {
```

Comment:
@minsunny can you explicitly set the ordinal on these. 
This is a long list and will continue to grow. If through mistake or bad merge one were to be added 'in-between' an existing pair, then the ordinals will be off.

Reply:
@jlhughes updated